--- glpk-4.44/configure.ac.orig	2010-06-03 10:00:00.000000000 +0200
+++ glpk-4.44/configure.ac	2010-10-16 15:05:32.707743558 +0200
@@ -117,9 +117,9 @@
       LIBMYSQL="libmysqlclient.dylib"
       ;;
    *)
-      LIBIODBC="libiodbc.so"
-      LIBODBC="libodbc.so"
-      LIBMYSQL="libmysqlclient.so"
+      LIBIODBC="libiodbc.so.2"
+      LIBODBC="@libodbc_soname@"
+      LIBMYSQL="@libmysqlclient_soname@"
       ;;
 esac
 
